dev/table.h

```diff
@@ -26,28 +26,31 @@ namespace sqlite_orm {
              */
             std::string name;
 
-            bool _without_rowid = false;
+            const bool _without_rowid;
```

Comment:
are you sure that we need this field? I think that we have already enough data to evaluate whether table has PK if we know storage type. And we don't need more data to store

Reply:
Ok, i will remove this field